docs/kandel/how-does-kandel-work/parameters.md

```diff
@@ -20,6 +20,6 @@ Initial inventory | The initial inventory is the amount of base tokens and quote
 Current price | The current price of the base token that is used for constructing the price distribution.<br /><br />*Example: the price of ETH is used for the ETH/USDC pair*
 Price distribution | It is a distribution of price levels/grid points within a selected price range. Kandel starts from the *Min price*, all the way up to the *Max price*. It is calculated using geometric progression and a key metric called "ratio": <br /><br />*Price at index i = min_price * ratio ^i*
 Volume distribution | It defines how liquidity is spread throughout the price grid (previously defined as the price distribution in the [visual step-by-step](./step-by-step-visual-explanation))
-Step size | It is the distance between an executed bid/ask and its dual offer. In the case of an offer is taken at price p :<br />• Lowest price = 1000 USDC per ETH<br />• Highest price = 1500 USDC per ETH<br /><br />*Example: if step size=1, it means that if a bid is taken at an index i, the ask ([dual offer](LINK TO GLOSSARY)) is posted at an index i+1*
-[Bounty](../../terms/bounty) | It is the required amount of native tokens to be deposited into the strategy. The [Bounty](../../terms/bounty) is required to post an offer on Mangrove; native tokens are used to cover failing offers.<br /><br />It covers the whole price grid, hence:<br />• *Kandel bounty = bounty per offer * number of offers*<br /><br />Example: if the selected pair is on the Polygon network, the native token provision would be an amount of MATIC tokens.*
+Step size | It is the distance between an executed bid/ask and its dual offer. In the case of an offer is taken at price p :<br />• Lowest price = 1000 USDC per ETH<br />• Highest price = 1500 USDC per ETH<br /><br />*Example: if step size=1, it means that if a bid is taken at an index i, the ask ([dual offer](../../terms/dual-offer.md)) is posted at an index i+1*
+[Provision](../../terms/provision.md) | It is the required amount of native tokens to be deposited into the strategy. A [provision](../../terms/provision.md) is required to post an offer on Mangrove; native tokens are used to cover failing offers.<br /><br />It covers the whole price grid, hence:<br />• *Kandel bounty = bounty per offer * number of offers*<br /><br />Example: if the selected pair is on the Polygon network, the native token provision would be an amount of MATIC tokens.*

```

Comment:
Kandel provision = provision per offer * number of offers * 2
so; provision instead of bounty.
and times 2, because in theory all bids and asks could be live at the same time if offers are sniped, in which case we need provisions for all at once - so it is a conservative bound.